pkg/cstorvolume/v1alpha1/build.go

```diff
@@ -0,0 +1,274 @@
+// Copyright © 2018-2019 The OpenEBS Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import (
+	apis "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
+	errors "github.com/openebs/maya/pkg/errors/v1alpha1"
+)
+
+// Builder is the builder object for CStorVolume
+type Builder struct {
+	cstorvolume *CStorVolume
+	errs        []error
+}
+
+// NewBuilder returns new instance of Builder
+func NewBuilder() *Builder {
+	return &Builder{cstorvolume: &CStorVolume{object: &apis.CStorVolume{}}}
+}
+
+// WithName sets the Name field of CStorVolume with provided value.
+func (b *Builder) WithName(name string) *Builder {
+	if len(name) == 0 {
+		b.errs = append(
+			b.errs,
+			errors.New("failed to build cstorvolume object: missing cstorvolume name"),
+		)
+		return b
+	}
+	b.cstorvolume.object.Name = name
+	return b
+}
+
+// WithGenerateName sets the GenerateName field of
+// CStorVolume with provided value
+func (b *Builder) WithGenerateName(name string) *Builder {
+	if len(name) == 0 {
+		b.errs = append(
+			b.errs,
+			errors.New("failed to build cstorvolume object: missing cstorvolume generateName"),
```

Comment:
Pease review the message. It should be uniform across.

Reply:
done